wave_vision/CMakeLists.txt

```diff
@@ -4,40 +4,52 @@ PROJECT(wave_vision)
 # DEPENDENCIES
 FIND_PACKAGE(OpenCV REQUIRED)
 FIND_PACKAGE(Eigen3 REQUIRED)
+FIND_PACKAGE(Boost 1.40 COMPONENTS system filesystem REQUIRED)
```

Comment:
I have no problem with using Boost, the question is why 1.40? It implies our software will work with any version >= 1.40 which I doubt is true (we test 1.54).

I think we should maintain as few scripts as possible.